### DISCUSSION TITLE
Unintentional POST /add-verified-command request during e2e tests execution

### DISCUSSION THREAD

**Discussion by edwardzikthegreat** [cypress-io/cypress#27232] [Questions and Help]
After updating Cypress from v10 to v12.17.0 I started to get hundreds of those POST .../__cypress/add-verified-command requests with some args, key, name on url in payload... I noticed huge performance drop during tests execution (cypress open mode) - probably because of it... also now the Network traffic is unreadable :/ Can I turn it off?

**Comment by edwardzikthegreat** [comment]
Change which introduce it was implemented under the https://github.com/cypress-io/cypress/blame/a3e4a8a6067531ffcb1f85598bee9e3d69810fd1/packages/server/lib/routes-e2e.ts#L106

**Comment by Dragod** [comment]
> After updating Cypress from v10 to v12.17.0 I started to get hundreds of those POST .../__cypress/add-verified-command requests with some args, key, name on url in payload... I noticed huge performance drop during tests execution (cypress open mode) - probably because of it... also now the Network traffic is unreadable :/ Can I turn it off?

I was wondering this as well the network tab is really annoying with all those calls. Is there a a way to turn them off?

Edit: Actually filtering in the network tab with " -status-code:204"  does that but it's kind of annoying  I need to apply the same filter everytime
